remoulade/scheduler/scheduler.py

```diff
@@ -108,6 +108,9 @@ def decode(cls, data: bytes) -> "ScheduledJob":
 
 
 class Scheduler:
+
+    stopped = True
```

Comment:
Why put it here, I think it should be an instance variable and not a class variable